### PR TITLE
Add status tag to 'Check and assess' task

### PR DESCRIPTION
**Pull request by allbecauseyoutoldmeso**
### Description of change

Add a status tag to the 'Check and assess' task.

### Story Link

https://trello.com/c/PaYsVcQH/1130-create-page-structure-for-assessment-of-pd-stories

### Screenshot

Screenshot 2022-10-06 at 15 57 52

### CODE REVIEW

File: spec/system/planning_applications/summary_of_works_spec.rb [hunk 1]
```diff
@@ -84,9 +85,14 @@
       within("#assessment-information-tasks") do
         expect(page).to have_content("In progress")
       end
```
Comment (benbaumann95) [inline spec/system/planning_applications/summary_of_works_spec.rb:87]: I guess we may not need this specs if you are doing `expect(check_and_assess_list_item).to have_content("In progress")` anyway?

Comment (allbecauseyoutoldmeso) [reply spec/system/planning_applications/summary_of_works_spec.rb:87]: Surely you can't check too many tags 🤣

Comment (benbaumann95) [reply spec/system/planning_applications/summary_of_works_spec.rb:87]: There's a few examples of these scattered around. I guess this one is checking extra that it's within the id`"#assessment-information-tasks"`

File: spec/support/system_spec_helpers.rb [hunk 2]
```diff
@@ -8,4 +8,8 @@ def row_with_content(content, element = page)
   def selected_govuk_tab
     find("div[class='govuk-tabs__panel']")
   end
+
+  def check_and_assess_list_item
```
Comment (benbaumann95) [inline spec/support/system_spec_helpers.rb:12]: Is it worth making this dynamic / reusable ?

i.e. passing in the text as an arg so we can apply this to other list items

Comment (benbaumann95) [reply spec/support/system_spec_helpers.rb:12]:
```ruby
def list_item(text)
  find("li", text: text)
end
```

**Review by benbaumann95 — APPROVED** [review]

File: spec/system/planning_applications/summary_of_works_spec.rb (hunk 1 above)
Comment (benbaumann95) [reply spec/system/planning_applications/summary_of_works_spec.rb:87]: Ahh my bad ok lol

File: spec/support/system_spec_helpers.rb (hunk 2 above)
Comment (allbecauseyoutoldmeso) [reply spec/support/system_spec_helpers.rb]: Oh, excellent thought, yes! Especially because I've previously used the same thing elsewhere. DRY 🎉

**Review by EGiataganas — APPROVED** [review]

**Pull request merged**